src/Components/Components/test/Routing/RouteTableFactoryTests.cs

```diff
@@ -71,7 +71,7 @@ private class MyComponent : ComponentBase
         public void CanDiscoverRoutes_WithInheritance()
         {
             // Arrange & Act
-            var routes = RouteTableFactory.Create(new[] { typeof(MyComponent), typeof(MyInheritedComponent), });
+            var routes = RouteTableFactory.Create(new List<Type> { typeof(MyComponent), typeof(MyInheritedComponent), });
```

Comment:
Curious about why these changes

Reply:
I should undo these. The linker does not flow suppressions to compiler generated state machines. The previous code relied on Linq + yield return which resulted in warnings in the compiler generated code that I did not know how to suppress.